tests/xtro-sharpie/common-Network.ignore

```diff
@@ -0,0 +1,28 @@
+# the following protocols ae binded manualy as NativeObject
```

Comment:
incorrect comment, they are **not** bound as protocol, but types

I thought the protocols were in addition tp the types...